src/main/java/kotlinx/reflect/lite/builtins/KotlinBuiltIns.kt

```diff
@@ -0,0 +1,45 @@
+package kotlinx.reflect.lite.builtins
+
+import kotlinx.reflect.lite.name.*
+
+internal object StandardNames {
+    val BUILT_INS_PACKAGE_FQ_NAME = FqName("kotlin")
+
+    val FQ_NAMES = FqNames()
+    class FqNames {
+        val array = FqName("kotlin.Array")
```

Comment:
Yes, I'll add the note

Reply:
Please do, it will make reviewing the PR a lot easier. I'll easily see what is just a copy-paste of code that may be boilerplate-y is just used for its facilities, and what is the actual crux of the solution.

Reply:
This is more relevant than ever. I'm quite lost in the codebase now.